### ISSUE TITLE
Collections and multi photo posts

### ISSUE THREAD

**Issue by sirber** [pixelfed/pixelfed#5280]
In a collection, I can only add the first photo of a multi photo post. It would be nice that multi photo posts be counted as multiple photos. Thanks!

**Comment by sirber** [comment]
I'll fork the project, set it up and have a look at it.

**Comment by oculos** [comment]
That's true. It would be also nice to be able to add pictures to a collection from the post itself.

**Comment by ghost** [comment]
What are collections supposed to be? I haven't found anything in the documentation, but to my understanding, they're collections of posts, and not photos. To support my argument:
1. Clicking on an item in a collection leads to the post; it isn't a direct link to the picture.
2. All the code and database structure seems to be built this way: a collection item is a post, not an individual photo.

But then there's a minor issue that's bugging my eyes: in the header of a collection, the counter says "photos" instead of "posts". This could be potentially misleading.

@dansup Is my assumption correct? If so, I'd gladly send in a pull request, it seems to be a one-liner.

**Comment by sirber** [comment]
It's based on the Mastodon API, so it's a post with multiple photos.
